include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -855,6 +925,38 @@ class FEEvaluationBase
    */
   unsigned int cell;
 
+  /**
+   * Flag holding information whether a face is on the left or the right of a
+   * cell. Not used for cells.
+   **/
+  bool is_interior_face;
+
+  /**
+   * Stores the index an FEFaceEvaluation object is currently pointing into
+   * (minus face, plus face, face associated with cell).
+   */
+  unsigned int face_vector_access_index;
+
+  /**
+   * Stores the current number of a face within the given cell in case
+   * `is_face==true`, using values between `0` and `2*dim`.
+   */
+  unsigned int face_no;
+
+  /**
+   * Stores the orientation of the given face with respect to the standard
+   * orientation, 0 if in standard orientation.
+   */
+  unsigned int face_orientation;
+
+  /**
+   * Stores the subface index of the given face. Usually, this variable takes
+   * the value -1 to indicate integration over the full face, but in case the
```

Comment:
> value -1 

don't we have a `const static unsigned int` for that, instead of hard-coding `-1`?